packages/test/test-service-load/src/test/mini.spec.ts

```diff
@@ -0,0 +1,120 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import child_process from "child_process";
+import commander from "commander";
+import { TestDriverTypes } from "@fluidframework/test-driver-definitions";
+import { ILoadTestConfig } from "../testConfigFile";
+import { createTestDriver, getProfile, initialize, safeExit } from "../utils";
+
+async function main() {
```

Comment:
not sure if this is just for testing, but you shouldn't re-implement anything here. you should just run the current orchestrator via a child process:
https://nodejs.org/api/child_process.html